docs/pages/API/Text/index.js

```diff
@@ -64,11 +58,11 @@ const apiOptions = [{
    description: 'Pass a style <code>object</code> to customize inline style of the component',
    responsive: false
 }, {
-   name: 'uppercase',
-   type: 'Boolean',
-   default: '<code>false</code>',
-   description: 'Set to <code>true</code> to apply the css equivalent of <code>text-transform: uppercase;</code>',
-   responsive: false
+   name: 'transform',
+   type: 'String',
+   default: '<code>normal</code>',
+   description: 'Set text transformations for Text. Any compatible css <code>text-transform</code> values are valid, including: <code>capitalize</code>, <code>lowercase</code>, <code>uppercase</code>',
```

Comment:
Do *all* css compatible values work, including `initial`, `inherit`, and `none`?